Algorithm Codes/Graph/AdjacencyListGraph.py

```diff
@@ -150,6 +150,34 @@ def floyd_warshall(self):
                 return None
         
         return dist
+    
+    def prims(self):
+        nodes_left = self.n
+        visited = [False for i in range(self.n)]
+        visited[0] = True
+        edges = []
+
+        heap = []
+
+        curr = self.adj[0].head
+        while(curr is not None):

```

Comment:
```suggestion
        while(curr):
```

A minor change according to PEP8 :)

Reply:
https://peps.python.org/pep-0008/#programming-recommendations
says to use is not none

Reply:
Oh you are right! 
I will drop my suggestion :) 

Thanks!

Reply:
I noticed your way should be suitable for this case.
I did a quick research and this is also filtering `[ ]` and `0` and others. So `is not None` should be right :) 

I will just remove the parenthesis